src/core.c/Any.rakumod

```diff
@@ -145,7 +145,9 @@ my class Any { # declared in BOOTSTRAP
 
     multi method iterator(Any:) { self.list.iterator }
 
-    method match(Any:U: |) { self.Str; nqp::getlexcaller('$/') = Nil }
+    proto method match(|)         {*}
+    multi method match(Any:U: |)  { self.Str; nqp::getlexcaller('$/') = Nil }
```

Comment:
Meh, I should have looked closer, sorry.

I don't think the `nqp::getlexcaller` will actually affect the caller's `$/`, because that is hidden by the proto, at least on the non-MoarVM backends.

See: https://github.com/rakudo/rakudo/blob/main/src/core.c/Str.rakumod#L1495-L1500

Reply:
Also I wonder whether the `proto` in `Str` can now go?

Reply:
Good catches, I didn't want to rock the boat so I just left the previous method body as is.

Reply:
Removing the `proto` in `Str` unfortunately causes spectests to stall out.

Reply:
It happens right after `t/spec/integrations/weird-errors.t` returns `ok`. A handful of tests have also failed by then.

Reply:
It does look like removing the `nqp::getlexcaller` caller does not have any negative impacts, at least as far as Roast coverage goes.